specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json

```diff
@@ -1445,14 +1456,18 @@
               {
                 "value": "P-256K",
                 "description": "The SECG SECP256K1 elliptic curve."
+              },
+              {
+                "value": "Ed25519",
+                "description": "The Ed25519 Edwards-curve."
```

Comment:
Nit: when not used as a compound adjective, it's just "Edwards curve". RFCs most often use it as two distinct words as well.

Reply:
If you change it, there's a couple instances throughout. Please be sure to update them all.

Reply:
Will update in next iteration.